.github/workflows/gradle.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
We should also set up builds for Java 11 and Windows. Windows is important since Windows file systems are a pain in the ass.

https://github.com/bndtools/bnd/blob/50e451283ce51cbe8366504bbe12915015b64d13/.github/workflows/cibuild.yml#L91-L106

Reply:
I used a matrix to test on windows, mac and linux with java 8 and 11